integration-app/src/main/java/uk/gov/crowncommercial/esourcing/integration/service/TenderApiService.java

```diff
@@ -275,21 +334,22 @@ private Rfxs getRfxsBody(
     RfxSetting rfxSetting = new RfxSetting();
 
     rfxSetting.setTenderReferenceCode(tenderReferenceCode);
-    rfxSetting.setShortDescription(String.valueOf(rfx.getShortDescription()));
-
+    rfxSetting.setShortDescription(tenderReferenceCode + " " + rfx.getShortDescription());
+    rfxSetting.setLongDescription(rfx.getLongDescription());
     BuyerCompany buyerCompany = new BuyerCompany();
     buyerCompany.setId(defaultBuyerCompanyId);
     rfxSetting.setBuyerCompany(buyerCompany);
 
     OwnerUser ownerUser = new OwnerUser();
-    ownerUser.setLogin(defaultOwnerUser);
+
+      ownerUser.setLogin(ownerLogin);
+
     rfxSetting.setOwnerUser(ownerUser);
-    rfxSetting.setValue(String.valueOf(rfx.getValue()));
+    rfxSetting.setValue(rfx.getValue());
 
-    if(rfx.getRfiFlag() != null && rfx.getRfiFlag().equals("1"))
-    {
+    if (rfx.getRfiFlag() != null && rfx.getRfiFlag().equals("1")) {
```

Comment:
Is rfiFlag an integer or a String? It may have changed recently an so the comparison is no longer correct? Might be better to change type to a primitive?

Reply:
Yeah it changed from string to integer